diff -Nur a/dracut.sh b/dracut.sh
--- a/dracut.sh	2022-02-18 11:32:53.000000000 +0000
+++ b/dracut.sh	2022-09-07 15:01:59.055893269 +0100
@@ -1054,11 +1054,11 @@
         outfile="$dracutsysrootdir$efidir/Linux/linux-$kernel${MACHINE_ID:+-${MACHINE_ID}}${BUILD_ID:+-${BUILD_ID}}.efi"
     else
         if [[ -e $dracutsysrootdir/boot/vmlinuz-$kernel ]]; then
-            outfile="/boot/initramfs-$kernel.img"
+            outfile="/boot/initrd-$kernel"
         elif [[ $MACHINE_ID ]] && { [[ -d $dracutsysrootdir/boot/${MACHINE_ID} ]] || [[ -L $dracutsysrootdir/boot/${MACHINE_ID} ]]; }; then
             outfile="$dracutsysrootdir/boot/${MACHINE_ID}/$kernel/initrd"
         else
-            outfile="$dracutsysrootdir/boot/initramfs-$kernel.img"
+            outfile="$dracutsysrootdir/boot/initrd-$kernel"
         fi
     fi
 fi
