diff -Nur a/Makefile b/Makefile
--- a/Makefile	2021-09-27 22:48:05.000000000 +0100
+++ b/Makefile	2022-09-21 12:52:08.248406011 +0100
@@ -63,12 +63,11 @@
 install:
 	$(INSTALL_DIR) $(DESTDIR)/etc
 	cp -pPR etc/* etc.$(OS)/* $(DESTDIR)/etc/
-	echo "Gentoo Base System release ${PV}" > ${DESTDIR}/etc/gentoo-release
 	$(INSTALL_DIR) $(DESTDIR)/lib
 	cp -pPR lib.$(OS)/* $(DESTDIR)/lib/
 	$(INSTALL_DIR) $(DESTDIR)/usr/lib
 	ln -snf ../usr/lib/os-release ${DESTDIR}/etc/os-release
-	./make_os_release ${OS} ${PV} > $(DESTDIR)/usr/lib/os-release
+	./make_os_release ${OS} rolling > $(DESTDIR)/usr/lib/os-release
 	$(INSTALL_DIR) $(DESTDIR)/usr/share/baselayout
 	cp -pPR share.$(OS)/* $(DESTDIR)/usr/share/baselayout/
 
diff -Nur a/make_os_release b/make_os_release
--- a/make_os_release	2021-09-27 22:48:05.000000000 +0100
+++ b/make_os_release	2022-09-21 12:49:21.760390518 +0100
@@ -4,12 +4,12 @@
 	exit 1
 fi
 cat <<EOF
-NAME=Gentoo
-ID=gentoo
-PRETTY_NAME="Gentoo ${1}"
+NAME=Redcore
+ID=redcore
+PRETTY_NAME="Redcore ${1} Hardened"
 ANSI_COLOR="1;32"
-HOME_URL="https://www.gentoo.org/"
-SUPPORT_URL="https://www.gentoo.org/support/"
-BUG_REPORT_URL="https://bugs.gentoo.org/"
+HOME_URL="https://redcorelinux.org/"
+SUPPORT_URL="https://wiki.redcorelinux.org/"
+BUG_REPORT_URL="https://bugs.redcorelinux.org/"
 VERSION_ID="${2}"
 EOF
