diff -Nur a/os-probes/mounted/common/90linux-distro b/os-probes/mounted/common/90linux-distro
--- a/os-probes/mounted/common/90linux-distro	2021-01-28 00:23:50.827441813 +0000
+++ b/os-probes/mounted/common/90linux-distro	2021-01-28 00:27:33.379716742 +0000
@@ -98,6 +98,15 @@
 elif [ -e "$dir/etc/gentoo-release" ]; then
 	short="Gentoo"
 	long="$(cat "$dir/etc/gentoo-release")"
+elif [ -e "$dir/etc/redcore-release" ]; then
+	short="Redcore"
+	long="$(cat "$dir/etc/redcore-release")"
+elif [ -e "$dir/etc/kogaion-release" ]; then
+	short="Kogaion"
+	long="$(cat "$dir/etc/kogaion-release")"
+elif [ -e "$dir/etc/argent-release" ]; then
+	short="Argent"
+	long="$(cat "$dir/etc/argent-release")"
 elif [ -e "$dir/etc/cobalt-release" ]; then
 	short="Cobalt"
 	long="$(cat "$dir/etc/cobalt-release")"
