diff -Nur a/src/modules/localecfg/main.py b/src/modules/localecfg/main.py
--- a/src/modules/localecfg/main.py	2020-05-11 19:53:23.000000000 +0100
+++ b/src/modules/localecfg/main.py	2020-05-21 16:11:50.013845904 +0100
@@ -149,20 +149,6 @@
     target_locale_conf_path = "{!s}/etc/locale.conf".format(install_path)
     target_etc_default_path = "{!s}/etc/default".format(install_path)
 
-    # restore backup if available
-    if os.path.exists(target_locale_gen_bak):
-        shutil.copy2(target_locale_gen_bak, target_locale_gen)
-        libcalamares.utils.debug("Restored backup {!s} -> {!s}"
-            .format(target_locale_gen_bak, target_locale_gen))
-
-    # run locale-gen if detected; this *will* cause an exception
-    # if the live system has locale.gen, but the target does not:
-    # in that case, fix your installation filesystem.
-    if os.path.exists('/etc/locale.gen'):
-        rewrite_locale_gen(target_locale_gen, target_locale_gen, locale_conf)
-        libcalamares.utils.target_env_call(['locale-gen'])
-        libcalamares.utils.debug('{!s} done'.format(target_locale_gen))
-
     # write /etc/locale.conf
     with open(target_locale_conf_path, "w") as lcf:
         for k, v in locale_conf.items():
