diff -Nur a/steam.desktop b/steam.desktop
--- a/steam.desktop	2018-09-27 19:20:20.000000000 +0100
+++ b/steam.desktop	2020-01-06 00:00:32.785536584 +0000
@@ -27,7 +27,7 @@
 Comment[tr]=Steam üzerinden oyun oynama ve düzenleme uygulaması
 Comment[uk]=Програма для керування іграми та запуску ігор у Steam
 Comment[vi]=Ứng dụng để quản lý và chơi trò chơi trên Steam
-Exec=/usr/bin/steam %U
+Exec=/usr/bin/redcore-steam %U
 Icon=steam
 Terminal=false
 Type=Application
@@ -63,7 +63,7 @@
 Name[tr]=Mağaza
 Name[uk]=Крамниця
 Name[vi]=Cửa hàng
-Exec=steam steam://store
+Exec=redcore-steam steam://store
 
 [Desktop Action Community]
 Name=Community
@@ -93,7 +93,7 @@
 Name[tr]=Topluluk
 Name[uk]=Спільнота
 Name[vi]=Cộng đồng
-Exec=steam steam://url/SteamIDControlPage
+Exec=redcore-steam steam://url/SteamIDControlPage
 
 [Desktop Action Library]
 Name=Library
@@ -123,7 +123,7 @@
 Name[tr]=Kütüphane
 Name[uk]=Бібліотека
 Name[vi]=Thư viện
-Exec=steam steam://open/games
+Exec=redcore-steam steam://open/games
 
 [Desktop Action Servers]
 Name=Servers
@@ -153,7 +153,7 @@
 Name[tr]=Sunucular
 Name[uk]=Сервери
 Name[vi]=Máy chủ
-Exec=steam steam://open/servers
+Exec=redcore-steam steam://open/servers
 
 [Desktop Action Screenshots]
 Name=Screenshots
@@ -183,7 +183,7 @@
 Name[tr]=Ekran Görüntüleri
 Name[uk]=Скріншоти
 Name[vi]=Ảnh chụp
-Exec=steam steam://open/screenshots
+Exec=redcore-steam steam://open/screenshots
 
 [Desktop Action News]
 Name=News
@@ -213,7 +213,7 @@
 Name[tr]=Haberler
 Name[uk]=Новини
 Name[vi]=Tin tức
-Exec=steam steam://open/news
+Exec=redcore-steam steam://open/news
 
 [Desktop Action Settings]
 Name=Settings
@@ -243,11 +243,11 @@
 Name[tr]=Ayarlar
 Name[uk]=Налаштування
 Name[vi]=Thiết lập
-Exec=steam steam://open/settings
+Exec=redcore-steam steam://open/settings
 
 [Desktop Action BigPicture]
 Name=Big Picture
-Exec=steam steam://open/bigpicture
+Exec=redcore-steam steam://open/bigpicture
 
 [Desktop Action Friends]
 Name=Friends
@@ -277,4 +277,4 @@
 Name[tr]=Arkadaşlar
 Name[uk]=Друзі
 Name[vi]=Bạn bè
-Exec=steam steam://open/friends
+Exec=redcore-steam steam://open/friends
